CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## Unpublished
+
+**Added**
```

Comment:
This list should be more complete with respect to actual features/API added. See the Java SDK's CHANGELOG for an example